backend/services/implementations/classService.ts

```diff
@@ -191,6 +193,32 @@ class ClassService implements IClassService {
     }
   }
 
+  async archiveClass(id: string): Promise<ClassResponseDTO> {
+    let archivedClass: Class | null;
+    try {
+      archivedClass = await MgClass.findOneAndUpdate(
+        { _id: id },
+        { isActive: false },
```

Comment:
i think we have to use the `$set` operator here? 

see more under `update` parameter: https://www.mongodb.com/docs/manual/reference/method/db.collection.findOneAndUpdate/

<img width="443" alt="Screenshot 2023-05-29 at 11 52 49 AM" src="https://github.com/uwblueprint/jump-math/assets/41273486/3acc9ffd-3f6a-4bcb-911d-5c3831e52870">

let me know if i'm misinterpreting this!